test-cases/longevity/longevity-mv-synchronous-updates-12h.yaml

```diff
@@ -0,0 +1,30 @@
+test_duration: 780
+prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=10000200  -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(5)' -pop seq=1..10000200 -log interval=15",
+                    "cassandra-stress user profile=/tmp/mv_synchronous_updates.yaml ops'(insert=25)' cl=QUORUM n=20000000 -mode cql3 native -rate threads=100",
+                    "cassandra-stress user profile=/tmp/mv_synchronous_updates_lwt.yaml ops'(insert=25)' cl=QUORUM n=1000000 -mode cql3 native -rate threads=100"
+                   ]
+stress_read_cmd: ["cassandra-stress mixed cl=QUORUM duration=690m  -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=100 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..10000200 -log interval=15",
+                  "cassandra-stress user profile=/tmp/mv_synchronous_updates.yaml ops'(select_base=3,select_mv=3,select_mv_2=3,url_column_update=1,row_delete=1)' cl=QUORUM duration=690m -mode cql3 native -rate threads=50",
+                  "cassandra-stress user profile=/tmp/mv_synchronous_updates.yaml ops'(select_base=3,select_mv=3,select_mv_2=3,url_column_update=1,row_delete=1)' cl=QUORUM duration=690m -mode cql3 native -rate threads=50",
+                  "cassandra-stress user profile=/tmp/mv_synchronous_updates.yaml ops'(select_base=3,select_mv=3,select_mv_2=3,url_column_update=1,row_delete=1)' cl=QUORUM duration=690m -mode cql3 native -rate threads=50",
+                  "cassandra-stress user profile=/tmp/mv_synchronous_updates_lwt.yaml ops'(select_base=1,select_mv=1,select_mv_2=1,lwt_deletes=10)' cl=QUORUM duration=690m -mode cql3 native -rate threads=50",
+                  "cassandra-stress user profile=/tmp/mv_synchronous_updates_lwt.yaml ops'(select_base=1,select_mv=1,select_mv_2=1,lwt_update_one_column=10)' cl=QUORUM duration=690m -mode cql3 native -rate threads=50",
+                  "cassandra-stress user profile=/tmp/mv_synchronous_updates_on_standard.yaml ops'(select=6, mv_pk_update_pk=2, mv_pk_update_non_pk=2)' cl=QUORUM  duration=690m -pop seq=0..1000200 -mode cql3 native -rate threads=50",
+                  "cassandra-stress user profile=/tmp/mv_synchronous_updates_on_standard.yaml ops'(select=6, mv_pk_update_pk=2, mv_pk_update_non_pk=2)' cl=QUORUM  duration=690m -pop seq=0..1000200 -mode cql3 native -rate threads=50",
+                 ]
+run_fullscan: ['{"mode": "random", "ks_cf": "random", "interval": 15}'] # 'ks.cf|random, interval(min)'
```

Comment:
I think you don't need it here, since you have enough scans in your profiles.

Reply:
removed